src/zjs_sensor.c

```diff
@@ -406,7 +403,22 @@ static void zjs_sensor_onstop_c_callback(void *h, void *argv)
     send.data.sensor.channel = handle->channel;
 
     ZVAL reading_val = jerry_create_null();
-    zjs_set_property(obj, "reading", reading_val);
+    jerry_value_t null_val = jerry_create_null();
+    switch(handle->channel) {
+    case SENSOR_CHAN_ACCEL_XYZ:
+    case SENSOR_CHAN_GYRO_XYZ: ;
```

Comment:
this is actually intentional, if you don't put a semi-colon there, you get a compilation warning
error: a label can only be part of a statement and a declaration is not a statement
         double x = ((double *)reading)[0];

so i'll probably just declare it at the top